### ISSUE TITLE
how to handle 'register_buffer' with multiple gpu cards

### ISSUE THREAD

**Issue by eeric** [mk-minchul/AdaFace#15]
self.register_buffer('batch_mean', torch.ones(1)*(20))
        self.register_buffer('batch_std', torch.ones(1)*100)

**Comment by mk-minchul** [comment]
Multi-GPU is handled automatically in our code with the relevant config `gpus`. The code has been tested on multi-gpu settings.

**Comment by eeric** [comment]
thanks.
I was also do as following：
in forward function
self.batch_mean = self.batch_mean.to(device), such as device = logits.device
